### PR TITLE
Lab cryptol overview tweaks

### PR DESCRIPTION
**Pull request by AssemblyZig**
Results of review and modification of cryptol overview lab.

### CODE REVIEW

File: labs/cryptol-overview/overview.md [hunk 1]
```diff
@@ -1,53 +1,18 @@
 # Introduction
 
-Cryptol is a domain specific language and tool suite created by Galois, Inc with support from NSA cryptographers. The language has lots of cool programming language features that make it well suited for applications in high assurance systems and cryptographic development, including:
+Cryptol is a domain specific language and tool suite created by Galois, Inc, with support from NSA cryptographers. The language has lots of cool programming language features that make it well suited for applications in high assurance systems and cryptographic development, including:
```
Comment (weaversa) [inline labs/cryptol-overview/overview.md:3]: How about "Galois, Inc.,"

File: labs/cryptol-overview/overview.md [hunk 2]
```diff
@@ -227,25 +192,59 @@ Q.E.D.
...
+  where
+    greeting = "Hello, " # name
+```
 
-Furthermore, Cryptol provides direct access and easily integrates with powerful tools such as SMT solvers and the Software Analysis Workbench (SAW). These tools allow the user to *prove* facts and demonstrate properties about their code which can provide assurance guarantees that go far beyond simple unit testing.
+If you have Cryptol installed, you should be able to do the following from the terminal:
 
-We will introduce some of these features below and discuss how they support building Cryptographic specifications and evaluations. If you have access to the Cryptol interpreter you can follow along with some of the examples, but a detailed introduction to the Cryptol interpreter will be provided in future lessons.
+```shell
+$ cryptol
```
Comment (weaversa) [inline labs/cryptol-overview/overview.md:212]: I suggest we make this lab literate and remove the sayHello.cry file. I'm happy to take a stab at it.

**Comment by weaversa** [comment]
rebased to the lab-overview pull request

**Comment by weaversa** [comment]
@AssemblyZig Do you have more suggestions, or can we merge into Jacob's in-progress branch?

**Comment by AssemblyZig** [comment]
All sounds good to me. For some reason, I didn't see any of these updates
until my laptop was rebooted. I am not a fan of technology, and the feeling
seems to be mutual.

Cheers,
Zig

**Pull request merged**